website/content/index.mdx

```diff
@@ -0,0 +1,202 @@
+    </a>
+  </div>
+</div>
+
+## What is Armada?
+
+Armada is a multi-Kubernetes cluster batch job meta-scheduler designed to handle massive-scale workloads. Built on top of Kubernetes, Armada enables organizations to distribute millions of batch jobs per day across tens of thousands of nodes spanning multiple clusters, making it an ideal solution for high-throughput computational workloads.
+
+Armada serves as middleware that transforms Kubernetes into a powerful batch processing platform while maintaining compatibility with service workloads. It addresses the fundamental limitations of running batch workloads at scale on Kubernetes by providing:
+
+- **Multi-cluster orchestration**: Schedule jobs across many Kubernetes clusters seamlessly
+- **High-throughput queueing**: Handle millions of queued jobs using specialized storage layers
+- **Advanced batch scheduling**: Fair queuing, gang scheduling, preemption, and resource limits
+- **Enterprise-grade reliability**: Secure, highly available components designed for production use
+
+As a [CNCF Sandbox project](https://www.cncf.io/), Armada is actively maintained and used in production environments, including at [G-Research](https://www.gresearch.com/) where it processes millions of jobs daily.
+
+## Why Use Armada?
+
+### Kubernetes Limitations for Batch Workloads
+
+Traditional Kubernetes faces several challenges when running batch workloads at scale:
+
+1. **Single Cluster Scaling Limits**: Scaling a single Kubernetes cluster beyond a certain size is [challenging](https://openai.com/blog/scaling-kubernetes-to-7500-nodes/), typically maxing out around 5,000-15,000 nodes depending on configuration.
+
+2. **Storage Backend Constraints**: Achieving very high throughput using etcd, Kubernetes' in-cluster storage backend, is [challenging](https://etcd.io/docs/v3.5/op-guide/performance/) and can become a bottleneck for job queuing.
+
+3. **Inadequate Batch Scheduling**: The default [kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) lacks essential batch scheduling features like fair queuing, gang scheduling, and intelligent preemption.
+
+### Armada's Solution
+
+Armada overcomes these limitations by:
+
+- **Distributing across multiple clusters**: Manage thousands of nodes across many Kubernetes clusters
+- **Out-of-cluster scheduling**: Perform queueing and scheduling using specialized storage layers
+- **Purpose-built batch scheduler**: Include advanced scheduling features designed specifically for batch workloads
+
+## Key Features and Benefits
+
+### Core Scheduling Features
+
+**Fair-Use Scheduling**
+
+- Maintains fair resource share over time across users and teams
+- Based on dominant resource fairness principles
+- Includes priority factors for different queues
+- Inspired by HTCondor priority systems
+
+**High Throughput Processing**
+
+- Handle millions of queued jobs simultaneously
+- Specialized storage layer optimized for batch workloads
```

Comment:
I am not sure about this part, it isn't intuitive to me what is `Specialized storage layer`